tests/scenario_quality_test_stock_lot_production.rst

```diff
@@ -0,0 +1,186 @@
+========================
```

Comment:
I think you can merge this test with scenario_quality_test_stock_lot.rst

Reply:
We need to ensure that extras depend works properly